src/sysroot.rs

```diff
@@ -1,166 +0,0 @@
-//! Download and manage sysroots.
```

Comment:
I noticed this file was unused which confused me for a bit, so figured I'd go ahead and clean it up!